content/article/geolytica.md

```diff
@@ -0,0 +1,69 @@
+{
+    "title"       : "Yet Another Perl-Powered Company: Geolytica",
+    "url"         : "/geolytica-powered-by-perl",
+    "authors"     : ["ervin-ruci"],
+    "date"        : "2024-12-31T12:00:00",
+    "tags"        : ["geolytica", "geoparsing", "ai", "geocoding", "poi", "geodata"],
+    "draft"       : false,
+    "image"       : "/images/geolytica-powered-by-perl/camel.png",
+    "thumbnail"   : "/images/geolytica-powered-by-perl/camel.png",
+    "description" : "Celebrating 20 years of innovation at Geolytica, Ervin Ruci shares his journey with Perl, from building geocoding engines to enhancing OpenStreetMap data.",
+    "categories"  : "perl"
+}
+
+
+## Who am I?
+
+[Ervin Ruci](https://eruci.com). I've been using Perl since 1998, building
+student information systems at Mount Allison University, then building registry
+systems for cira.ca between 2000 and 2005 - the year I quit my day job and
+became a location independent entrepreneur.
+
+My current active companies listed at [Geolytica.com](https://geolytica.com)
```

Comment:
I'd lose the first sentence and make the rest of this paragraph the first one. First you need to hook them, then you can explain who you are. 

Most of this section might do better at the end in the "about the author" section. Get to the meat quickly. :)